druid-derive/tests/data.rs

```diff
@@ -102,3 +102,28 @@ fn test_data_derive_same() {
     assert!(v.same(&v));
     assert!(!v.same(&TypeParamForUserTraitAndLifetimeEnum::V1(Value(12))));
 }
+
+#[derive(Data, Clone)]
+struct DataAttrEq {
+    #[data(eq)]
```

Comment:
what happens if this *also* includes a `same_fn` annotation?

Reply:
the same_fn will be ignored. It is probably a better idea to panic/show error